### ISSUE TITLE
Documentation update suggestion for Troubleshooting.md in docs

### ISSUE THREAD

**Issue by marktomlinson1** [terraform-google-modules/terraform-example-foundation#613]
### TL;DR

Cloud Build triggers for 1-org fail with no errors/log entries if 'git add .' is executed in wrong folder. Suggest adding this to Troubleshooting list. 

### Expected behavior

git push --set-upstream origin plan (step 16) and git push origin production (step 18) of Cloud Build instructions for 1-org succesfully initiate builds

### Observed behavior

git push --set-upstream origin plan (step 16) and git push origin production (step 18) of Cloud Build instructions for 1-org result in failed builds with no errors / log entries displayed. 

Problem was caused by running 'git add .' (Step 15) in the envs/shared folder, and as a consequence the cloudbuild-tf-* files had not been added to the source repo and couldn't be found by the triggers. 

### Terraform Configuration

```hcl
As per 1-org
```

### Terraform Version

```sh
Terraform v0.13.0
```

### Additional information

Following Cloud Build instructions in this instance.
